src/DCDataSet.cpp

```diff
@@ -36,13 +38,52 @@
 
 namespace splash
 {
+    herr_t H5Ewalk_strings(unsigned n, const H5E_error2_t *err_desc, void *client_data)
```

Comment:
you should give this another name that does not start with `H5E`. This looks like it is coming from HDF5 itself.